.github/workflows/ci.yml

```
@@ -1,141 +1,141 @@
+          - name: Node 22
+            NODE_VERSION: 22.12.0
+      fail-fast: false
+    name: ${{ matrix.name }}
+    timeout-minutes: 15
+    runs-on: ubuntu-latest
+    env:
+      NODE_VERSION: ${{ matrix.NODE_VERSION }}
+    steps:
+      - name: Determine major node version
+        id: node
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2116:style:2:37: Useless echo? Instead of 'cmd $(echo foo)', just use 'cmd foo'

(shellcheck)